pkg/sql/conn_executor.go

```diff
@@ -2289,15 +2291,18 @@ func (ex *connExecutor) execCmd() (retErr error) {
 		}
 
 	case PrepareStmt:
+		isExtendedProtocolCmd = true
```

Comment:
My recommendation would be to add the assignment to every `case` (assign false as needed) so that if any gets copy-paste d in the future the person is forced to think about whether to set this.

An alternative is to make a map type -> bool and assert in a test or init method that all the types have a decision taken about whether they are extended protocol or not.

Reply:
i like the map idea, thanks!